generate-stackbrew-library.sh

```diff
@@ -100,6 +100,15 @@ for version; do
 		variantParent="$(awk 'toupper($1) == "FROM" { print $2 }' "$dir/Dockerfile")"
 		variantArches="${parentRepoToArches[$variantParent]}"
 
+		if [ "$variant" = 'bookworm' ] && [ "$version" != '5.1' ] && [ "$version" != '6.0' ]; then
+			# https://github.com/docker-library/redmine/pull/393 (6.1+ deps need newer rust on other arches and just isn't worth the trouble)
+			variantArches="$(jq -r -R <<<"$variantArches" '
```

Comment:
Sorry, I should've made this a separate suggestion, or at least mentioned it explicitly in my text. :sweat_smile:

```suggestion
			variantArches="$(jq <<<"$variantArches" --raw-input --raw-output '
```

Reply:
"What if suggestions were actually rendered just exactly like PR diffs" and other wild ideas